operations/cancellationpolicies.md

```diff
@@ -102,6 +100,197 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+    },
+    {
+      "Id": "3858905e-32fd-4ea3-9b3d-5cf93014745a",
+      "RateId": "c5bb9d3e-849d-4119-a6bc-ac87a2b20810",
+      "Applicability": "Creation",
+      "FeeExtent": [
+        "TimeUnits",
+        "Products"
+      ],
+      "ApplicabilityOffset": "0M7D0:0:0.0",
+      "FeeMaximumTimeUnits": 0,
+      "AbsoluteFee": {
+        "Currency": "EUR",
+        "Value": 30
+      },
+      "RelativeFee": 0
+    },
+    {
+      "Id": "1d7097a9-058e-44fe-9db0-f8cd2726e1ff",
+      "RateId": "1e7509b2-ff94-48c0-bdf7-3fd277fe72fe",
+      "Applicability": "Creation",
+      "FeeExtent": [
+        "TimeUnits",
+        "Products"
+      ],
+      "ApplicabilityOffset": "0M0D0:0:0.0",
+      "FeeMaximumTimeUnits": 10,
+      "AbsoluteFee": null,
+      "RelativeFee": 1
+    }
+  ],
+  "Cursor": "769fc613-838f-41a7-ac2a-aff100c3189f"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CancellationPolicies` | array of [Cancellation Policy](cancellationpolicies.md#cancellation-policy) | required, max 1000 items | The filtered cancellation policies. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest cancellation policy returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older cancellation policies. |
+
+#### Cancellation Policy
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the cancellation policy. |
+| `RateGroupId` | string | required | Unique identifier of the rate group the cancellation policy belongs to. |
+| `Applicability` | [Cancellation Policy Applicability](cancellationpolicies.md#cancellation-policy-applicability) | required | Applicability mode of the cancellation policy. |
+| `FeeExtent` | array of [Cancellation Fee Extent](cancellationpolicies.md#cancellation-fee-extent) | required | Extent for the cancellation fee, i.e. what should be in scope for the automatic payment. |
+| `ApplicabilityOffset` | string | required | Offset for order start (assuming Applicability is set to Start) from which the fee is applied. |
+| `FeeMaximumTimeUnits` | integer | optional | Maximum number of time units the cancellation fee is applicable to. |
+| `AbsoluteFee` | [Currency value (ver 2023-02-02)](_objects.md#currency-value-ver-2023-02-02) | required | Absolute value of the fee. |
+| `RelativeFee` | number | required | Relative value of the fee, as a percentage of the reservation price. |
+| `IsActive` | boolean | required | Whether the cancellation policy is still active. |
+
+## Get cancellation policies for Reservations
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Given the IDs of Reservations, returns Cancellation Policies that has been locked for those reservations.
+
+### Request
+
+`[PlatformAddress]/api/connector/v2/cancellationPolicies/get`
```

Comment:
Will need a bit different Url

Reply:
You actually don't need a separate operation, add `ReservationIds` parameter to `cancellationPolicies/getAll`

Reply:
Also start, end dates would have to be empty when reservationIds are passed in